README.md

```diff
@@ -31,8 +31,13 @@ TODO
 All examples assume a client and/or channel has been created as follows:
 
 ```go
-client := ably.NewRestClient(ably.ClientOptions{ApiKey: "xxxx"})
-channel := client.Channel('test')
+options, err := ably.NewClientOptions("xxx:xxx")
```

Comment:
Out of interest, if ClientOptions is effectively just a data type, do you think this style is necessary?  Could we not simply do the err checking in the Client instead and keep ClientOptions as a simple immutable set of options used to instance a Client?